google/resource_bigtable_instance.go

```diff
@@ -132,27 +132,13 @@ func resourceBigtableInstanceRead(d *schema.ResourceData, meta interface{}) erro
 
 	defer c.Close()
 
-	instances, err := c.Instances(ctx)
+	instance, err := c.InstanceInfo(ctx, d.Id())
 	if err != nil {
-		return fmt.Errorf("Error retrieving instances. %s", err)
+		return fmt.Errorf("Error retrieving instance. Could not find %s.", d.Id())
```

Comment:
You'll want to also remove the instance from state, like in https://github.com/terraform-providers/terraform-provider-google/blob/master/google/provider.go#L270 (realized I forgot about that in the first review, whoops!)

Also, might it be worth it to still pass along the error message from the bigtable client?